src/services/AccommodationService.js

```diff
@@ -0,0 +1,86 @@
+import { likes, accommodations } from '../database/models';
+import CommonQueries from './CommonQueries';
+
+/**
+ * @export
+ * @class AccommodationService
+ */
+class AccommodationService {
+  /**
+ * user POST likes/dislikes accommodation facility
+ * @static
+ * @param {object} req pass request body
+ * @memberof AccommodationService
+ * @returns {object} either an error or data
+ */
+  static async addAccommodationLike(req) {
+    let setObject, displayMessage;
+    const { like } = req.query;
+    const { accommodationId } = req.params;
+    const { id } = req.user;
+
+    const accommodation = await CommonQueries.findOne(accommodations, { where: { id: accommodationId } });
+    const isLikedOrDisliked = await CommonQueries.findOne(likes, { where: { userId: id } });
```

Comment:
You are repeatedly creating similar objects to query data. I have a feeling that you can use one function to return a desired object and achieve the same.

``` js
    const commonObject = (column, data) => {
      const query = {
        where: { 
            column: data
        }
      };
      return query;
    };

```